lightning_pose/data/utils.py

```diff
@@ -16,17 +16,25 @@ class BaseLabeledExampleDict(TypedDict):
     """Return type when calling __getitem__() on BaseTrackingDataset."""
 
     images: Union[
-        TensorType["RGB":3, "image_height", "image_width"],
-        TensorType["frames", "RGB":3, "image_height", "image_width"],
+        TensorType["RGB":3, "image_height", "image_width", float],
+        TensorType["frames", "RGB":3, "image_height", "image_width", float],
     ]
-    keypoints: TensorType["num_targets"]
+    keypoints: TensorType["num_targets", float]
     idxs: int
 
 
 class HeatmapLabeledExampleDict(BaseLabeledExampleDict):
     """Return type when calling __getitem__() on HeatmapTrackingDataset."""
 
-    heatmaps: TensorType["num_keypoints", "heatmap_height", "heatmap_width"]
+    heatmaps: TensorType["num_keypoints", "heatmap_height", "heatmap_width", float]
+
+class MultiviewHeatmapLabeledExampleDict(BaseLabeledExampleDict):
```

Comment:
make this inherit from `MultiviewLabeledExampleDict`, and add heatmaps (just copying `HeatmapLabeledExampleDict` for now). We'll add some more info to this later down the line.